extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -485,6 +495,7 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
 
   private renderReplySuccessMimeAttachments = (attachmentsFilenames: string[]) => {
     const attachments = $('<div id="attachments"></div>');
+    // eslint-disable-next-line @typescript-eslint/no-for-in-array
     for (const index in attachmentsFilenames) {
       if (attachmentsFilenames.hasOwnProperty(index)) {
```

Comment:
probably we can just replace `for in` with `for (const [index, filename] of attachmentsFilenames.entries()) {`

Reply:
Yeah, just missed it.